.github/workflows/dependencies/nvcc11-3.sh

```diff
@@ -41,7 +41,8 @@ sudo apt-get install -y          \
     cuda-nvml-dev-11-3           \
     cuda-nvtx-11-3               \
     libcufft-dev-11-3            \
-    libcurand-dev-11-3
+    libcurand-dev-11-3           \
+    libcusparse-dev-11-3
```

Comment:
@WeiqunZhang @atmyers Does this mean that installation instructions for e.g. Perlmutter should be updated to make sure that `cusparse` is installed?
https://warpx.readthedocs.io/en/latest/install/hpc/perlmutter.html

Reply:
It's part of the cuda package already installed on perlmutter. I don't we need anything special.